torch/_torch_docs.py

```diff
@@ -2509,6 +2509,12 @@ def merge_dicts(*dicts):
 Keyword args:
     {out}
 
+.. note::  This function is similar to SciPy's `scipy.special.digamma`.
+
+.. note::  From version 1.8 onwards, the digamma function returns `NaN` for non-positive integers,
+           while for `0`, it returns `-Inf` to be consistent with SciPy and C++ Standard.
+           Prior to version 1.8, the function would return `NaN` for non-positive integers and `0`.
```

Comment:
"and \`0\`' for ... ?

Reply:
Sorry didn't understand this one.

Reply:
I mean, the last sentence seems to end very abruptly. It says that zero is returned, but it never says what values would return zero.

Reply:
> the function would return `NaN` for non-positive integers and `0`.

What we want to phrase is that function used to return `nan` for non-positive integers as well as `0`.

Should I change "and `0`" to  "as well as for 0"?

Reply:
Oh I see what you're saying. I think I was confused because doesn't "nonpositive integers" include zero? See, for example: https://mathworld.wolfram.com/NonpositiveInteger.html.

Reply:
This would also suggest that the first use of "nonpositive integers" in this paragraph should be replaced with "negative integers".

Reply:
Oh right. Thanks!!!